addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -158,19 +166,18 @@ export class Many2ManyTagsField extends Component {
     }
 
     async deleteTag(id) {
-        const tagRecord = this.props.record.data[this.props.name].records.find(
+        let tagRecord = this.props.record.data[this.props.name].records.find(
             (record) => record.id === id
         );
-        await this.props.record.data[this.props.name].forget(tagRecord);
+
+        if (tagRecord) {
```

Comment:
I'm not sure to understand why/how you could remove a tag that isn't in the relation